### PR TITLE
Libnftnl

### PR DESCRIPTION
**Pull request by nvinson**

### CODE REVIEW

File: net-firewall/nftables/nftables-0.6-r5.ebuild [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	newinitd "${FILESDIR}"/${PN}.init ${PN}
+	keepdir /var/lib/nftables
+
+	systemd_dounit "${FILESDIR}"/systemd/${PN}-restore.service
+	systemd_enable_service basic.target ${PN}-restore.service
+}
+
+pkg_postinst() {
+	local save_file
+	save_file="${EROOT}var/lib/nftables/rules-save"
```
Comment (SoapGentoo) [inline net-firewall/nftables/nftables-0.6-r5.ebuild]: `${EROOT%/}/`

File: net-libs/libnftnl/libnftnl-1.0.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		$(use_with json json-parsing)
+}
+
+src_install() {
+	default
+	gen_usr_ldscript -a nftnl
+	prune_libtool_files
+
+	if use examples; then
+		find examples/ -name 'Makefile*' -delete
```
Comment (SoapGentoo) [inline net-libs/libnftnl/libnftnl-1.0.7.ebuild]: missing `|| die`

File: net-libs/libnftnl/libnftnl-1.0.7.ebuild [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+		dodoc -r examples/
```
Comment (SoapGentoo) [inline net-libs/libnftnl/libnftnl-1.0.7.ebuild]: remove the trailing slash (yes, not a nitpick, POSIX undefined)

File: net-libs/libnftnl/libnftnl-1.0.7.ebuild [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+src_configure() {
+	econf \
+		$(use_enable static-libs static) \
+		$(use_with json json-parsing)
+}
+
+src_install() {
+	default
+	gen_usr_ldscript -a nftnl
+	prune_libtool_files
```
Comment (SoapGentoo) [inline net-libs/libnftnl/libnftnl-1.0.7.ebuild]: could you replace this, so we can get rid of eutils? https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives

**Comment by nvinson** [comment]
I just saw bug 603228, and I would like to delay this request, so the fix can be added to the 0.7 bump.  Thanks.

**Comment by nvinson** [comment]
Final set of changes made.  I'm good with this being reviewed and merged now.

File: net-libs/libnftnl/libnftnl-1.0.7.ebuild [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+	find "${D}" -name '*.la' -delete || die "Could not rm libtool files"
+
+	if use examples; then
+		find examples/ -name 'Makefile*' -delete || die "Could not rm examples"
+		dodoc -r examples
+		docompress -x /usr/share/doc/${PF}/examples
+	fi
+}
+
+src_test() {
```
Comment (SoapGentoo) [inline net-libs/libnftnl/libnftnl-1.0.7.ebuild]: put `src_test` before `src_install`

**Comment by Natolumin** [comment]
nftables 0.7 requires libnftnl>=1.0.7, which removes xml support. Building this with the xml useflag set fails as the ebuild forces use of libnftnl=1.0.6 which is not compatible:

> configure: error: Package requirements (libnftnl >= 1.0.7) were not met:
> Requested 'libnftnl >= 1.0.7' but version of libnftnl is 1.0.6

**Comment by robbat2** [comment]
Merged

**Pull request closed without merging**